Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx

```diff
@@ -124,7 +124,7 @@ FLOW.AssignmentEditView = FLOW.ReactComponentView.extend(
         onSubmit: this.saveSurveyAssignment,
         handleSurveySelect: this.handleSurveySelect,
         handleDeviceCheck: this.handleDeviceCheck,
-        handleSelectAllDevice: this.handleSelectAllDevice,
+        // handleSelectAllDevice: this.handleSelectAllDevice,
```

Comment:
why has this been commented out? We can no longer select all devices?

Reply:
No, It's not being used. Select all devices is now handled by the `handleDeviceCheck` method

Reply:
then it would be better to remove it